pkgs/stdenv/adapters.nix

```diff
@@ -121,9 +121,6 @@ rec {
     # Apple does not provide a static version of libSystem or crt0.o
     # So we can’t build static binaries without extensive hacks.
     ++ lib.optional (!stdenv.hostPlatform.isDarwin) makeStaticBinaries
-
-    # Glibc doesn’t come with static runtimes by default.
```

Comment:
https://github.com/NixOS/nixpkgs/pull/134923#discussion_r693082882

Reply:
> was added in https://github.com/NixOS/nixpkgs/commit/3c36dccd7ca4bcc3a3899e794864780dc7a4ce8c and has ben unused since
makeStaticBinaries already adds glibc.static

Please write commit message bodies as proper sentences: capitalize first word and finish with a period.